pkg/daemon/ceph/client/pool.go

```diff
@@ -232,14 +236,28 @@ func givePoolAppTag(context *clusterd.Context, namespace string, poolName string
 }
 
 func setCommonPoolProperties(context *clusterd.Context, pool cephv1.PoolSpec, namespace, poolName, appName string) error {
-	compressionModeProperty := "compression_mode"
-	if pool.CompressionMode != "" {
+	if pool.Replicated.IsTargetRatioEnabled() {
+		err := SetPoolProperty(context, namespace, poolName, targetSizeRatioProperty, strconv.FormatFloat(pool.Replicated.TargetSizeRatio, 'f', -1, 32))
```

Comment:
For the properties that can now be set in the poolProperties, for backward compatibility instead of calling `SetPoolProperty()` here, how about just appending it to the PoolProperties?